modules/apps/analytics/analytics-client-js/src/plugins/dxp.js

```diff
@@ -54,7 +59,41 @@ function dxp(analytics) {
 		});
 	}
 
+	/**
+	 * Checks based on the URL param if it is a configuration portlet
+	 */
+	function isConfigurationPortlet() {
+		const searchParams = getSearchParams();
+
+		const portletId = searchParams.get(PARAM_PORTLET_ID_KEY);
+
+		if (portletId === PARAM_CONFIGURATION_PORTLET_NAME) {
+			return true;
+		}
+
+		return false;
```

Comment:
Can we just return `portletId === PARAM_CONFIGURATION_PORTLET_NAME`?

Reply:
This is a pattern we have to comply in Java code, if you guys don't, awesome :wink: